tags.json

```
@@ -3958,15 +3982,18 @@
     "park",
     "nature"
   ],
-  "trello": [],
+  "trello": [
+    "logo",
```

Remove trailing coma, json don't like that
```suggestion
    "logo"
```